### PR TITLE
refactor: use MenuListTile in AboutPage

### PR DESCRIPTION
**Pull request by britannio**
- Closes https://github.com/WorldHealthOrganization/app/issues/1250
- Moves the `menuItem` component out of the settings page and into the components folder(renamed to `MenuListTile`).
- Replaces the `LinkTextSpan`s with vertically stacked `MenuListTile`s

#### Screenshots
<details>
<summary>Screenshots</summary>
Before:

image

After:
image

</details>

#### How did you test the change?
* [ ] iOS Simulator
* [ ] Android Emulator
* [ ] iOS Device
* [x] Android Device
* [ ] `curl` to a dev App Engine server

---

#### Please follow this checklist. Please check each appropriate box (put an 'x' or check it after creating the PR).

- [x] REQUIRED: Do you have an Issue **assigned to you** for this PR?
- [x] Provided detailed pull request description and a succinct title
- [x] Tested your changes, especially after any code review iterations.
- [x] Included any relevant screenshots of UI updates.
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).
- [x] Verified all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE) file in the root of the repository.
- [x] Verified your name is in the [content/credits.yaml](https://github.com/WorldHealthOrganization/app/blob/master/content/credits.yaml) file (if you want it to be).

### CODE REVIEW

File: client/flutter/lib/pages/about_page.dart [hunk 1]
```diff
@@ -23,61 +23,50 @@ class AboutPage extends StatelessWidget {
...
+    }
+
+    void _openPrivacyPolicy(BuildContext context) {
+      final String url = S.of(context).legalLandingPagePrivacyPolicyLinkUrl;
+      launch(url);
+      FirebaseAnalytics().logEvent(name: 'PrivacyPolicy');
+    }
+
+    void _openLicenses(BuildContext context) {
+      Navigator.push(
```
Comment (advayDev1) [inline client/flutter/lib/pages/about_page.dart:39]: return this future

File: client/flutter/lib/pages/about_page.dart [hunk 2]
```diff
@@ -23,61 +23,50 @@ class AboutPage extends StatelessWidget {
...
 
+    void _openTermsOfService(BuildContext context) {
+      final String url = S.of(context).aboutPageTermsOfServiceLinkUrl;
+      launch(url);
+      FirebaseAnalytics().logEvent(name: 'TermsOfService');
+    }
+
+    void _openPrivacyPolicy(BuildContext context) {
+      final String url = S.of(context).legalLandingPagePrivacyPolicyLinkUrl;
+      launch(url);
```
Comment (advayDev1) [inline client/flutter/lib/pages/about_page.dart:34]: return this future

File: client/flutter/lib/pages/about_page.dart [hunk 3]
```diff
@@ -23,61 +23,50 @@ class AboutPage extends StatelessWidget {
         .of(context)
         .commonWorldHealthOrganizationCoronavirusCopyright(DateTime.now().year);
 
+    void _openTermsOfService(BuildContext context) {
+      final String url = S.of(context).aboutPageTermsOfServiceLinkUrl;
+      launch(url);
```
Comment (advayDev1) [inline client/flutter/lib/pages/about_page.dart:28]: return this future

**Comment by britannio** [comment]
> Let's make the padding between (c) and the divider more, maybe the same as the horizontal padding.
> 
> Screen Shot 2020-05-12 at 7 59 16 AM

Done, I'm not returning anything but I've "awaited" the futures you highlighted.

**Pull request merged**